.github/workflows/static-checks.yaml

```diff
@@ -188,6 +188,9 @@ jobs:
       - name: Install system dependencies
         run: |
           sudo apt-get -y install moreutils
+      - name: Install hunspell
+        run: |
+          sudo apt-get -y install hunspell
```

Comment:
Could you add it in the previous step, @GabyCT?

Reply:
As in:
```
sudo apt-get -y install hunspell moreutils
```

Reply:
@fidencio changes applied